src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -13,6 +13,11 @@ using namespace ::TerminalApp;
 using namespace winrt::Microsoft::Terminal::TerminalControl;
 using namespace winrt::TerminalApp;
 
+// {2bde4a90-d05f-401c-9492-e40884ead1d8}
+// uuidv5 properties: name format is UTF-16LE bytes
+static constexpr GUID c_terminalProfileNamespaceGuid =

```

Comment:
I am relatively sure this is the first usage of the hungarian `c_` in our codebase to denote constant. I thought we had a different style here. @adiviness

Reply:
To be fair, I grepped first ;p

Reply:
OK, I am often wrong. Perhaps it's used elsewhere too. I'm not that broken up by this. Won't fix if you like.

Reply:
We tend not to use the `c_` hungarian notation, though it is part of the Windows coding guidelines. In this case however is isn't needed because `c_` is for a global constant and this is a constant that doesn't leave this file so it's actually being applied incorrectly I believe.